### PR TITLE
Empty schema

### PR DESCRIPTION
**Pull request by avbelov23**
The name of the schema is an optional element in Transact-SQL

### CODE REVIEW

File: lib/sqli/sqli_parser.y [hunk 1]
```diff
@@ -118,6 +118,30 @@ sql_parens: sql_no_parens
...
+            sqli_store_data(ctx, &$data_name);
+        }
+        | data_name[sname] '.'[u1] data_name[tname] {
+            sqli_store_data(ctx, &$sname);
+            sqli_store_data(ctx, &$tname);
+            YYUSE($u1);
+        }
+        | data_name[dname] '.'[u1] schema_name_opt '.'[u2] data_name[tname] {
+            sqli_store_data(ctx, &$dname);
+            sqli_store_data(ctx, &$sname);
```
Comment (sorc1) [inline lib/sqli/sqli_parser.y]: `$sname` is not defined in the context.

**Comment by avbelov23** [comment]
DONE

File: lib/sqli/sqli_parser.y [hunk 2]
```diff
@@ -99,7 +99,7 @@ data_cont:
         | after_exp_cont
         ;
 
-update: TOK_UPDATE[tk1] colref_exact TOK_SET[tk2] expr_list {
+update: TOK_UPDATE[tk1] tableref TOK_SET[tk2] expr_list {
```
Comment (sorc1) [inline lib/sqli/sqli_parser.y]: According to https://docs.microsoft.com/ru-ru/sql/t-sql/queries/update-transact-sql?view=sql-server-2017 there is also a three-dots syntax:
```<object> ::=  
{   
    [ server_name . database_name . schema_name .   
    | database_name .[ schema_name ] .   
    | schema_name .  
    ]  
    table_or_view_name}
```
Moreover, how would look column references to such a table without an alias? Is it possible to use four dots style like below?
`server_name . database_name . schema_name . table_name . column_name`

In any case, what about to not have a special syntax for tableref but instead just to extend the colref_exact syntax? If you believe that a special syntax for tableref is necessary then I think we should name it as `tableref_exact`, not as `tableref`, to not mix it with existant `table_ref`.

**Comment by sorc1** [comment]
The changes are now in master branch. Thanks!

**Pull request closed without merging**